packages/patternfly-4/react-core/src/components/Tabs/Tabs.js

```diff
@@ -49,8 +50,24 @@ class Tabs extends React.Component {
   id = getUniqueId();
   tabList = React.createRef();
 
-  handleTabClick(event, eventKey) {
+  handleTabClick(event, eventKey, tabChild) {
     this.props.onSelect(event, eventKey);
+    // process any children outside of the component
+    if (tabChild) {
+      React.Children.map(this.props.children, (child, i) => {
+        document.getElementById(child.props.tabChild).hidden = true;
+      });
+      // most recently selected tabChild
+      document.getElementById(tabChild).hidden = false;
```

Comment:
We should test if the object returned is not undefined. A dev could potentially provide an incorrect ID.

Probably want to ensure we obtained a `TabChild` type as well. Otherwise, you cannot be certain the returned object has a `hidden` property.

Reply:
Added these checks in new commit; let me know if that's not what you're looking for. I'll add the refs in a separate commit.